apps/mark-scan/frontend/src/app_contest_ms_either_neither.test.tsx

```diff
@@ -68,151 +45,8 @@ const pickOneContest = find(
 assert(eitherNeitherContest.type === 'yesno');
 assert(pickOneContest.type === 'yesno');
 
-const ballotStyleId = '1';
 const precinctId = '6526';
 
-function expectPrintedVotes(
```

Comment:
All this testing was essentially just testing that the rendered ballot reflected various either / neither possibilities. These possibilities are even special cased in the BmdPaperBallot component so this sort of basic logic - does the ballot reflect the passed votes - is tested elsewhere.